frontend/app/controllers/Joiner.scala

```diff
@@ -138,6 +138,7 @@ trait Joiner extends Controller {
         eventId <- PreMembershipJoiningEventFromSessionExtractor.eventIdFrom(request)
         event <- EventbriteService.getBookableEvent(eventId)
       } yield {
+        EventbriteService.getService(event).wsMetrics.put("user-sent-to-eventbrite", 1)
```

Comment:
Should this be stored under another metric (so we can tell the difference between user thats are sent to Eventbrite and users that can purchase via the iframe? We can aggregate them later.